lib/export-scala-regmap.js

```diff
@@ -15,10 +15,10 @@ const getScalaRegisterBundleName = (register) => {
 };
 
 const getRegisterFields = register => {
-  return register.fields || [{bits: register.size}];
+  return register.fields || [{bitOffset: 0, bitWidth: register.size}];
```

Comment:
Just to double check, this means that all previous DUH documents with the `bits` field are no longer compatible with this version of duh-scala. This is probably worth noting in release notes so that if people are upgrading to the latest version of duh-scala with this included, they know that they need to update their DUH documents.

Reply:
yes old duh docs will no longer be compatible, including `duh-export-scala` since it has to emit an `OMRegisterMap`